pages/accountLists/[accountListId]/tools/csv.page.test.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import { render, waitFor } from '@testing-library/react';
+import TestRouter from '__tests__/util/TestRouter';
+import TestWrapper from '__tests__/util/TestWrapper';
+import { useApiConstants } from 'src/components/Constants/UseApiConstants';
+import { CsvImportViewStepEnum } from 'src/components/Tool/Import/Csv/CsvImportContext';
+import { get } from 'src/components/Tool/Import/Csv/csvImportService';
+import { useAccountListId } from 'src/hooks/useAccountListId';
+import theme from 'src/theme';
+import CsvHome from './csv.page';
+
+jest.mock('src/hooks/useAccountListId');
+jest.mock('src/components/Constants/UseApiConstants');
+jest.mock('src/components/Tool/Import/Csv/csvImportService');
+
+const accountListId = 'accountListId';
+const csvFileId = 'csvFileId';
+
+const constants = {
+  sendAppeals: [
+    { id: true, value: 'Yes' },
+    { id: false, value: 'No' },
+  ],
+};
+
+const buildRouter = (tab) => {
+  return {
+    isReady: true,
+    query: {
+      tab: tab,
+      id: csvFileId,
+    },
+    replace: jest.fn(),
+  };
+};
+
+const renderCsvHome = (router) => {
+  return render(
+    <TestWrapper>
+      <ThemeProvider theme={theme}>
+        <TestRouter router={router}>
+          <CsvHome />
+        </TestRouter>
+      </ThemeProvider>
+    </TestWrapper>,
+  );
+};
```

Comment:
Nothing crazy, just simplifying code.
```suggestion
const renderCsvHome = (router) => (
  render(
    <TestWrapper>
      <ThemeProvider theme={theme}>
        <TestRouter router={router}>
          <CsvHome />
        </TestRouter>
      </ThemeProvider>
    </TestWrapper>,
  );
);
```